### PR TITLE
Javascript typing tutor

### PR DESCRIPTION
**Pull request by dotkeenan**

### CODE REVIEW

File: javascript-typing-tutor/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Document</title>
+  <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/js/bootstrap.min.js"> -->
+  <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css"
+  integrity="sha512-P5MgMn1jBN01asBgU0z60Qk4QxiXo86+wlFahKrsQf37c9cro517WzVSPPV1tDKzhku2iJ2FVgL67wG03SGnNA=="
+  crossorigin="anonymous" type="text/css"/> -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css"
+    integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css" type="text/css">
+  <!-- <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script> -->
```
Comment (thebearingedge) [inline javascript-typing-tutor/index.html]: You can delete the commented out stuff.

File: javascript-typing-tutor/style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.words {
+  font-size: 1.5rem;
+}
+
+.red {
+  color: red;
+}
+
+.green {
+  /* color: rgb(0, 236, 0); */
```
Comment (thebearingedge) [inline javascript-typing-tutor/style.css]: Here too.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Sweet! Please add a GIF too so i can se it! 👀

**Comment by dotkeenan** [comment]
typing-tutor

Man it's been a while, I forgot about the screenshots! 🤦‍♂️

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**